source/examples/examples/recipes.md

```diff
@@ -8,15 +8,14 @@ Recipes show you how to test common scenarios in Cypress.
 {% fa fa-github %} {% url https://github.com/cypress-io/cypress-example-recipes %}
 
 ## Fundamentals 
-
 Recipe  | Description
 --- | --- 
 {% url 'Node Modules' https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/fundamentals__node-modules %} | Import your own Node modules
-{% url "Environment variables" https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/server-communication__env-variables %} | Passing environment variables to tests
```

Comment:
How come all the double quotes got changed to single quotes?

Reply:
There were a mixture and I figured I'd take the opportunity to clean it up. The other files in this folder seemed to be using single quotes.